.circleci/config.yml

```diff
@@ -26,7 +26,9 @@ jobs:
       - run:
           name: pytest
           when: always
-          command: python -m pytest
+          command: |
+            pip install --user pip==9.0.1
```

Comment:
why do we have this?

Is there a breakage with Circle CI?

If so, we need an issue about that, and this needs to be in a separate commit.

Reply:
Because CircleCI tests were failing, this is the case for almost all PRs where TravisCI is getting passed but not CircleCI. So, Makman2 suggested to tackle this.

Reply:
Should I make an issue and then a PR or make a separate commit in this PR?

Reply:
@jayvdb , in this PR: https://github.com/coala/coala/pull/5789 , although TravisCI build is passing but not CircleCI: https://circleci.com/gh/coala/coala/19361 . The reason being incompatible version of pip.

Reply:
@ayan-b , we have made CircleCI 'non voting' , so it doesnt need to be fixed for your PR to be merged.

Just remove this from your PR, and it can be merged.

fwiw, a PR should only solve one problem, unless there are multiple breakages which all need to be solved.  If that ever happens, we require separate commits for each problem.